.github/workflows/e2e-wpuf.yml

```
@@ -0,0 +1,122 @@
+    timeout-minutes: 60
+    runs-on: ubuntu-latest    
+    
+    #Cloning WPUF-Lite + WPUF-PRO
+    steps:
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: "7.3"
+            
+      - uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)